apps/src/bounce/BounceVisualizationColumn.jsx

```diff
@@ -1,7 +1,7 @@
 var msg = require('../locale');
 var commonStyles = require('../commonStyles');
 
-var GameButtons = require('../templates/GameButtons');
+var GameButtons = require('../templates/GameButtons').default;
```

Comment:
This feels a little bit gross. I wonder if it would actually be less gross to just mix requires and imports here, i.e. do
```
var commonStyles = require('../commonStyles');
import GameButtons from '../templates/GameButtons';
var ArrowButtons = require('../templatesArrowButtons');
```